### ISSUE TITLE
[Bug] Optional settings are not working on repeatable element

### ISSUE THREAD

**Issue by IoanAndrei97** [Laravel-Backpack/CRUD#3483]
# Bug report

I'm trying to add the "init_rows", "min_rows" and "max_rows" options on the repeatable element, but they are not working. The only options that's working is "new_item_label".

I also tried the documentation example, but it's not working neither.

### What I expected to happen
I expected to have a bigger/smaller init rows and the min/max rows to do what they have to do.

### What happened
Nothing. The options are ignored.

### PHP VERSION:
PHP 7.4.4 (cli) (built: Jul 27 2020 08:12:57) ( NTS )

### LARAVEL VERSION:
v8.21.0@a61cab167c35f465a923737ee6e6fb99cd5fde88

### BACKPACK VERSION:
4.1.30@62856b1f01fc3e05c80140fc22f85ea2aac8368c

**Comment by pxpm** [comment]
Hello @IoanAndrei97 

Sorry for this misleading, we are uneven merging the docs and the PR. 

We merged the docs at the same time we merged the PR into master branch, but master branch is not tagged yet (you are using 4.1.30, and this functionality will be available in 4.1.31 when it gets tagged).

I am going to close this, thanks for the report, I expect the tagging to happen during today, I will ping @tabacitu on this one!

Thanks,
Pedro
